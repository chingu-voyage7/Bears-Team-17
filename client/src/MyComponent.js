import React from 'react';
import PrimaryButton from './common/buttons/Primary';

const onclick = () => {
  console.log('clickety-click');
};

const MyComponent = () => (
  <div>
    <PrimaryButton onClick={onclick} >
      Click Me
    </PrimaryButton>
  </div>
);

export default MyComponent;

