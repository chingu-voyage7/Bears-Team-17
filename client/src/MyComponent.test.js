import React from 'react';
import { render } from 'react-testing-library';
import MyComponent from './MyComponent';

describe('MyComponent', () => {
  it('renders MyComponent', () => {
    const { container, getByText } = render(<MyComponent />);
    expect(container.firstChild).toMatchSnapshot();
    const text = getByText('Click Me');
    expect(text).toBeDefined();
  });
});

