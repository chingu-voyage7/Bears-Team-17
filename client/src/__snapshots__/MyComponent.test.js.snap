// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent renders MyComponent 1`] = `
<div>
  <button
    class="sc-bdVaJa jjCqhi"
  >
    Click Me
  </button>
</div>
`;
